modules/base/tool.go

```diff
@@ -31,7 +31,16 @@ import (
 	"github.com/gogits/gogs/modules/setting"
 )
 
-var Sanitizer = bluemonday.UGCPolicy().AllowAttrs("class").Matching(regexp.MustCompile(`[\p{L}\p{N}\s\-_',:\[\]!\./\\\(\)&]*`)).OnElements("code")
+func BuildSanitizer() (p *bluemonday.Policy) {
```

Comment:
No actually this line: https://github.com/gogits/gogs/pull/2406/files#diff-88e948dbecd728e2513ba9bafe876faaR40